### ISSUE TITLE
wish - date/time modification and file access rights

### ISSUE THREAD

**Issue by ildarkhasanshin** [icetee/remote-ftp#567]
of any atom of any sublime text does not appear in the list of date/time modification of files and file permissions
in fact often they need me, and other programmers, probably, too
this task is now partly solves only through atom-commander

**Comment by jimmaaay** [comment]
@ildarkhasanshin Sorry I'm a little confused with what you wrote. Are you making a feature request for something to show last date modified and file permissions of the remote files?

**Comment by ildarkhasanshin** [comment]
yes, probably need an additional command to display the last modification date of the file and the file permissions
